internal/shim-kvm/.cargo/config

```diff
@@ -1,10 +1,14 @@
 [build]
-target = "x86_64-unknown-linux-musl"
+# FIXME: "x86_64-unknown-none" is not yet a tier 2 target, so setting this as
+#        the default target breaks rust-analyzer.
+# target = "x86_64-unknown-none"
```

Comment:
I think we'd prefer to remove this whole file as we move towards being fully cargo workspace native.